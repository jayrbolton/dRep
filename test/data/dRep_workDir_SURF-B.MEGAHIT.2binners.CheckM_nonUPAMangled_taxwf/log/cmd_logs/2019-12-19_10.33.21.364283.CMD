prodigal -i /home/sumin/kbsdk-workspace/dRep/test/data/binned_contig_files_8bins/Bin.038.fasta -d /home/sumin/kbsdk-workspace/dRep/test/data/res.dRep.txwf.uniq/data/prodigal/Bin.038.fasta.fna -a /home/sumin/kbsdk-workspace/dRep/test/data/res.dRep.txwf.uniq/data/prodigal/Bin.038.fasta.faa -m -p meta
