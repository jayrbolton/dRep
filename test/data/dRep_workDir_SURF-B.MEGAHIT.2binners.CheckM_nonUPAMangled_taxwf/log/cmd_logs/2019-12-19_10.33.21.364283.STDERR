-------------------------------------
PRODIGAL v2.6.3 [February, 2016]         
Univ of Tenn / Oak Ridge National Lab
Doug Hyatt, Loren Hauser, et al.     
-------------------------------------
Request:  Metagenomic, Phase:  Training
Initializing training files...done!
-------------------------------------
Request:  Metagenomic, Phase:  Gene Finding
Finding genes in sequence #1 (12841 bp)...done!
Finding genes in sequence #2 (11976 bp)...done!
Finding genes in sequence #3 (3612 bp)...done!
Finding genes in sequence #4 (9290 bp)...done!
Finding genes in sequence #5 (5906 bp)...done!
Finding genes in sequence #6 (24106 bp)...done!
Finding genes in sequence #7 (2661 bp)...done!
Finding genes in sequence #8 (25704 bp)...done!
Finding genes in sequence #9 (21028 bp)...done!
Finding genes in sequence #10 (16852 bp)...done!
Finding genes in sequence #11 (16780 bp)...done!
Finding genes in sequence #12 (9257 bp)...done!
Finding genes in sequence #13 (9656 bp)...done!
Finding genes in sequence #14 (5831 bp)...done!
Finding genes in sequence #15 (4252 bp)...done!
Finding genes in sequence #16 (4837 bp)...done!
Finding genes in sequence #17 (5686 bp)...done!
Finding genes in sequence #18 (7005 bp)...done!
Finding genes in sequence #19 (4549 bp)...done!
Finding genes in sequence #20 (5244 bp)...done!
Finding genes in sequence #21 (8052 bp)...done!
Finding genes in sequence #22 (6679 bp)...done!
Finding genes in sequence #23 (11357 bp)...done!
Finding genes in sequence #24 (3103 bp)...done!
Finding genes in sequence #25 (4090 bp)...done!
Finding genes in sequence #26 (13172 bp)...done!
Finding genes in sequence #27 (15483 bp)...done!
Finding genes in sequence #28 (4192 bp)...done!
Finding genes in sequence #29 (6466 bp)...done!
Finding genes in sequence #30 (8334 bp)...done!
Finding genes in sequence #31 (5952 bp)...done!
Finding genes in sequence #32 (5197 bp)...done!
Finding genes in sequence #33 (5790 bp)...done!
Finding genes in sequence #34 (6030 bp)...done!
Finding genes in sequence #35 (18637 bp)...done!
Finding genes in sequence #36 (8217 bp)...done!
Finding genes in sequence #37 (15240 bp)...done!
Finding genes in sequence #38 (3432 bp)...done!
Finding genes in sequence #39 (3436 bp)...done!
Finding genes in sequence #40 (4220 bp)...done!
Finding genes in sequence #41 (4610 bp)...done!
Finding genes in sequence #42 (13631 bp)...done!
Finding genes in sequence #43 (7958 bp)...done!
Finding genes in sequence #44 (6028 bp)...done!
Finding genes in sequence #45 (9571 bp)...done!
Finding genes in sequence #46 (4170 bp)...done!
Finding genes in sequence #47 (7132 bp)...done!
Finding genes in sequence #48 (5145 bp)...done!
Finding genes in sequence #49 (7322 bp)...done!
Finding genes in sequence #50 (15668 bp)...done!
Finding genes in sequence #51 (3817 bp)...done!
Finding genes in sequence #52 (2866 bp)...done!
Finding genes in sequence #53 (27335 bp)...done!
Finding genes in sequence #54 (13519 bp)...done!
Finding genes in sequence #55 (10485 bp)...done!
Finding genes in sequence #56 (11567 bp)...done!
Finding genes in sequence #57 (13534 bp)...done!
Finding genes in sequence #58 (3817 bp)...done!
Finding genes in sequence #59 (7311 bp)...done!
Finding genes in sequence #60 (20251 bp)...done!
Finding genes in sequence #61 (9469 bp)...done!
Finding genes in sequence #62 (15832 bp)...done!
Finding genes in sequence #63 (3570 bp)...done!
Finding genes in sequence #64 (5115 bp)...done!
Finding genes in sequence #65 (4620 bp)...done!
Finding genes in sequence #66 (6422 bp)...done!
Finding genes in sequence #67 (3864 bp)...done!
Finding genes in sequence #68 (3548 bp)...done!
Finding genes in sequence #69 (3293 bp)...done!
Finding genes in sequence #70 (19124 bp)...done!
Finding genes in sequence #71 (7856 bp)...done!
Finding genes in sequence #72 (11694 bp)...done!
Finding genes in sequence #73 (10386 bp)...done!
Finding genes in sequence #74 (5151 bp)...done!
Finding genes in sequence #75 (5717 bp)...done!
Finding genes in sequence #76 (6768 bp)...done!
Finding genes in sequence #77 (9154 bp)...done!
Finding genes in sequence #78 (11529 bp)...done!
Finding genes in sequence #79 (3508 bp)...done!
Finding genes in sequence #80 (5675 bp)...done!
Finding genes in sequence #81 (2706 bp)...done!
Finding genes in sequence #82 (6673 bp)...done!
Finding genes in sequence #83 (33242 bp)...done!
Finding genes in sequence #84 (8462 bp)...done!
Finding genes in sequence #85 (8289 bp)...done!
Finding genes in sequence #86 (23825 bp)...done!
Finding genes in sequence #87 (3800 bp)...done!
Finding genes in sequence #88 (20828 bp)...done!
Finding genes in sequence #89 (9556 bp)...done!
Finding genes in sequence #90 (8771 bp)...done!
Finding genes in sequence #91 (12463 bp)...done!
Finding genes in sequence #92 (14760 bp)...done!
Finding genes in sequence #93 (35096 bp)...done!
Finding genes in sequence #94 (4761 bp)...done!
Finding genes in sequence #95 (9732 bp)...done!
Finding genes in sequence #96 (5796 bp)...done!
Finding genes in sequence #97 (3274 bp)...done!
Finding genes in sequence #98 (5970 bp)...done!
Finding genes in sequence #99 (4274 bp)...done!
Finding genes in sequence #100 (6913 bp)...done!
Finding genes in sequence #101 (5114 bp)...done!
Finding genes in sequence #102 (5456 bp)...done!
Finding genes in sequence #103 (6425 bp)...done!
Finding genes in sequence #104 (12681 bp)...done!
Finding genes in sequence #105 (5787 bp)...done!
Finding genes in sequence #106 (6231 bp)...done!
Finding genes in sequence #107 (4281 bp)...done!
Finding genes in sequence #108 (4200 bp)...done!
Finding genes in sequence #109 (10133 bp)...done!
Finding genes in sequence #110 (3892 bp)...done!
Finding genes in sequence #111 (5039 bp)...done!
Finding genes in sequence #112 (6071 bp)...done!
Finding genes in sequence #113 (21012 bp)...done!
Finding genes in sequence #114 (5388 bp)...done!
Finding genes in sequence #115 (6773 bp)...done!
Finding genes in sequence #116 (13927 bp)...done!
Finding genes in sequence #117 (4889 bp)...done!
Finding genes in sequence #118 (3239 bp)...done!
Finding genes in sequence #119 (24159 bp)...done!
Finding genes in sequence #120 (2580 bp)...done!
Finding genes in sequence #121 (4527 bp)...done!
Finding genes in sequence #122 (11392 bp)...done!
Finding genes in sequence #123 (11002 bp)...done!
Finding genes in sequence #124 (4887 bp)...done!
Finding genes in sequence #125 (6593 bp)...done!
Finding genes in sequence #126 (9475 bp)...done!
Finding genes in sequence #127 (7768 bp)...done!
Finding genes in sequence #128 (6642 bp)...done!
Finding genes in sequence #129 (6950 bp)...done!
Finding genes in sequence #130 (12250 bp)...done!
Finding genes in sequence #131 (2892 bp)...done!
Finding genes in sequence #132 (5757 bp)...done!
Finding genes in sequence #133 (4266 bp)...done!
Finding genes in sequence #134 (27738 bp)...done!
Finding genes in sequence #135 (3619 bp)...done!
Finding genes in sequence #136 (3611 bp)...done!
Finding genes in sequence #137 (7514 bp)...done!
Finding genes in sequence #138 (5546 bp)...done!
Finding genes in sequence #139 (6469 bp)...done!
Finding genes in sequence #140 (10620 bp)...done!
Finding genes in sequence #141 (8123 bp)...done!
Finding genes in sequence #142 (12147 bp)...done!
Finding genes in sequence #143 (3312 bp)...done!
Finding genes in sequence #144 (12282 bp)...done!
Finding genes in sequence #145 (3998 bp)...done!
Finding genes in sequence #146 (12474 bp)...done!
Finding genes in sequence #147 (20004 bp)...done!
Finding genes in sequence #148 (2697 bp)...done!
Finding genes in sequence #149 (3411 bp)...done!
Finding genes in sequence #150 (16101 bp)...done!
Finding genes in sequence #151 (34773 bp)...done!
Finding genes in sequence #152 (2637 bp)...done!
Finding genes in sequence #153 (7728 bp)...done!
Finding genes in sequence #154 (4543 bp)...done!
Finding genes in sequence #155 (6805 bp)...done!
Finding genes in sequence #156 (6955 bp)...done!
Finding genes in sequence #157 (11525 bp)...done!
Finding genes in sequence #158 (7131 bp)...done!
Finding genes in sequence #159 (4182 bp)...done!
Finding genes in sequence #160 (6635 bp)...done!
Finding genes in sequence #161 (9537 bp)...done!
Finding genes in sequence #162 (19266 bp)...done!
Finding genes in sequence #163 (4688 bp)...done!
Finding genes in sequence #164 (2642 bp)...done!
Finding genes in sequence #165 (3956 bp)...done!
Finding genes in sequence #166 (4868 bp)...done!
Finding genes in sequence #167 (6173 bp)...done!
Finding genes in sequence #168 (11015 bp)...done!
Finding genes in sequence #169 (5533 bp)...done!
Finding genes in sequence #170 (6526 bp)...done!
Finding genes in sequence #171 (5690 bp)...done!
Finding genes in sequence #172 (9893 bp)...done!
Finding genes in sequence #173 (17747 bp)...done!
Finding genes in sequence #174 (15226 bp)...done!
Finding genes in sequence #175 (4103 bp)...done!
Finding genes in sequence #176 (3330 bp)...done!
Finding genes in sequence #177 (28405 bp)...done!
Finding genes in sequence #178 (4648 bp)...done!
Finding genes in sequence #179 (15359 bp)...done!
Finding genes in sequence #180 (2735 bp)...done!
Finding genes in sequence #181 (3989 bp)...done!
Finding genes in sequence #182 (3476 bp)...done!
Finding genes in sequence #183 (19543 bp)...done!
Finding genes in sequence #184 (7084 bp)...done!
Finding genes in sequence #185 (7387 bp)...done!
Finding genes in sequence #186 (12137 bp)...done!
Finding genes in sequence #187 (17617 bp)...done!
Finding genes in sequence #188 (4201 bp)...done!
Finding genes in sequence #189 (16412 bp)...done!
Finding genes in sequence #190 (26343 bp)...done!
Finding genes in sequence #191 (54219 bp)...done!
Finding genes in sequence #192 (4034 bp)...done!
Finding genes in sequence #193 (3355 bp)...done!
Finding genes in sequence #194 (8580 bp)...done!
Finding genes in sequence #195 (18963 bp)...done!
Finding genes in sequence #196 (8446 bp)...done!
Finding genes in sequence #197 (12009 bp)...done!
Finding genes in sequence #198 (29207 bp)...done!
Finding genes in sequence #199 (6151 bp)...done!
Finding genes in sequence #200 (4224 bp)...done!
Finding genes in sequence #201 (9539 bp)...done!
Finding genes in sequence #202 (8309 bp)...done!
Finding genes in sequence #203 (7572 bp)...done!
Finding genes in sequence #204 (12737 bp)...done!
Finding genes in sequence #205 (30976 bp)...done!
Finding genes in sequence #206 (9750 bp)...done!
Finding genes in sequence #207 (19073 bp)...done!
Finding genes in sequence #208 (3821 bp)...done!
Finding genes in sequence #209 (7251 bp)...done!
Finding genes in sequence #210 (11812 bp)...done!
Finding genes in sequence #211 (6142 bp)...done!
Finding genes in sequence #212 (3370 bp)...done!
Finding genes in sequence #213 (3838 bp)...done!
Finding genes in sequence #214 (22554 bp)...done!
Finding genes in sequence #215 (5541 bp)...done!
Finding genes in sequence #216 (8460 bp)...done!
Finding genes in sequence #217 (7059 bp)...done!
Finding genes in sequence #218 (8893 bp)...done!
Finding genes in sequence #219 (22828 bp)...done!
Finding genes in sequence #220 (25306 bp)...done!
Finding genes in sequence #221 (10682 bp)...done!
Finding genes in sequence #222 (24010 bp)...done!
Finding genes in sequence #223 (22821 bp)...done!
Finding genes in sequence #224 (35165 bp)...done!
Finding genes in sequence #225 (9089 bp)...done!
Finding genes in sequence #226 (2917 bp)...done!
Finding genes in sequence #227 (16863 bp)...done!
Finding genes in sequence #228 (4405 bp)...done!
Finding genes in sequence #229 (5289 bp)...done!
Finding genes in sequence #230 (4616 bp)...done!
Finding genes in sequence #231 (10934 bp)...done!
Finding genes in sequence #232 (6177 bp)...done!
Finding genes in sequence #233 (3193 bp)...done!
Finding genes in sequence #234 (7334 bp)...done!
Finding genes in sequence #235 (7052 bp)...done!
Finding genes in sequence #236 (5570 bp)...done!
Finding genes in sequence #237 (2791 bp)...done!
Finding genes in sequence #238 (6731 bp)...done!
Finding genes in sequence #239 (17297 bp)...done!
Finding genes in sequence #240 (4100 bp)...done!
Finding genes in sequence #241 (6629 bp)...done!
Finding genes in sequence #242 (9013 bp)...done!
Finding genes in sequence #243 (7538 bp)...done!
Finding genes in sequence #244 (4440 bp)...done!
Finding genes in sequence #245 (5105 bp)...done!
Finding genes in sequence #246 (39503 bp)...done!
Finding genes in sequence #247 (2533 bp)...done!
Finding genes in sequence #248 (2720 bp)...done!
Finding genes in sequence #249 (27880 bp)...done!
Finding genes in sequence #250 (27987 bp)...done!
Finding genes in sequence #251 (3979 bp)...done!
Finding genes in sequence #252 (3725 bp)...done!
Finding genes in sequence #253 (9418 bp)...done!
Finding genes in sequence #254 (22601 bp)...done!
Finding genes in sequence #255 (4834 bp)...done!
Finding genes in sequence #256 (10125 bp)...done!
Finding genes in sequence #257 (13184 bp)...done!
Finding genes in sequence #258 (7016 bp)...done!
Finding genes in sequence #259 (2512 bp)...done!
Finding genes in sequence #260 (9066 bp)...done!
Finding genes in sequence #261 (6505 bp)...done!
Finding genes in sequence #262 (7514 bp)...done!
Finding genes in sequence #263 (7407 bp)...done!
Finding genes in sequence #264 (2993 bp)...done!
Finding genes in sequence #265 (5650 bp)...done!
Finding genes in sequence #266 (35113 bp)...done!
Finding genes in sequence #267 (7379 bp)...done!
Finding genes in sequence #268 (10004 bp)...done!
Finding genes in sequence #269 (3610 bp)...done!
Finding genes in sequence #270 (3069 bp)...done!
Finding genes in sequence #271 (14924 bp)...done!
Finding genes in sequence #272 (7001 bp)...done!
Finding genes in sequence #273 (3827 bp)...done!
Finding genes in sequence #274 (4742 bp)...done!
Finding genes in sequence #275 (7679 bp)...done!
Finding genes in sequence #276 (6390 bp)...done!
Finding genes in sequence #277 (19210 bp)...done!
Finding genes in sequence #278 (39694 bp)...done!
Finding genes in sequence #279 (7304 bp)...done!
Finding genes in sequence #280 (10561 bp)...done!
Finding genes in sequence #281 (10405 bp)...done!
Finding genes in sequence #282 (14645 bp)...done!
Finding genes in sequence #283 (21895 bp)...done!
Finding genes in sequence #284 (4881 bp)...done!
Finding genes in sequence #285 (3271 bp)...done!
Finding genes in sequence #286 (3004 bp)...done!
Finding genes in sequence #287 (8907 bp)...done!
Finding genes in sequence #288 (31732 bp)...done!
Finding genes in sequence #289 (3751 bp)...done!
Finding genes in sequence #290 (3703 bp)...done!
Finding genes in sequence #291 (18861 bp)...done!
Finding genes in sequence #292 (8076 bp)...done!
Finding genes in sequence #293 (18185 bp)...done!
Finding genes in sequence #294 (10429 bp)...done!
Finding genes in sequence #295 (5070 bp)...done!
Finding genes in sequence #296 (5930 bp)...done!
Finding genes in sequence #297 (12703 bp)...done!
Finding genes in sequence #298 (13757 bp)...done!
Finding genes in sequence #299 (8171 bp)...done!
Finding genes in sequence #300 (12556 bp)...done!
Finding genes in sequence #301 (3233 bp)...done!
Finding genes in sequence #302 (31891 bp)...done!
Finding genes in sequence #303 (7315 bp)...done!
Finding genes in sequence #304 (7980 bp)...done!
Finding genes in sequence #305 (27564 bp)...done!
Finding genes in sequence #306 (7744 bp)...done!
Finding genes in sequence #307 (2633 bp)...done!
Finding genes in sequence #308 (3132 bp)...done!
Finding genes in sequence #309 (26530 bp)...done!
Finding genes in sequence #310 (7901 bp)...done!
Finding genes in sequence #311 (6648 bp)...done!
Finding genes in sequence #312 (18547 bp)...done!
Finding genes in sequence #313 (7534 bp)...done!
Finding genes in sequence #314 (2869 bp)...done!
Finding genes in sequence #315 (5000 bp)...done!
Finding genes in sequence #316 (5778 bp)...done!
Finding genes in sequence #317 (21443 bp)...done!
Finding genes in sequence #318 (8381 bp)...done!
Finding genes in sequence #319 (5397 bp)...done!
Finding genes in sequence #320 (9560 bp)...done!
Finding genes in sequence #321 (4259 bp)...done!
Finding genes in sequence #322 (17071 bp)...done!
Finding genes in sequence #323 (11316 bp)...done!
Finding genes in sequence #324 (8188 bp)...done!
Finding genes in sequence #325 (17977 bp)...done!
Finding genes in sequence #326 (6904 bp)...done!
Finding genes in sequence #327 (6455 bp)...done!
Finding genes in sequence #328 (12280 bp)...done!
Finding genes in sequence #329 (5570 bp)...done!
Finding genes in sequence #330 (7836 bp)...done!
Finding genes in sequence #331 (2867 bp)...done!
Finding genes in sequence #332 (15856 bp)...done!
Finding genes in sequence #333 (4057 bp)...done!
Finding genes in sequence #334 (10146 bp)...done!
Finding genes in sequence #335 (27290 bp)...done!
Finding genes in sequence #336 (3531 bp)...done!
Finding genes in sequence #337 (2787 bp)...done!
Finding genes in sequence #338 (13880 bp)...done!
Finding genes in sequence #339 (15577 bp)...done!
Finding genes in sequence #340 (7109 bp)...done!
Finding genes in sequence #341 (10385 bp)...done!
Finding genes in sequence #342 (5574 bp)...done!
Finding genes in sequence #343 (7485 bp)...done!
